base/dict.jl

```diff
@@ -223,7 +223,7 @@ function rehash!(h::Dict{K,V}, newsz = length(h.keys)) where V where K
     h.count = count
     h.ndel = 0
     h.maxprobe = maxprobe
-    @assert h.age == age0
+    h.age == age0 || error("versions of the dict do not match")
```

Comment:
You've left this change in from the previous PR.

Reply:
Well, what to do about it?

Reply:
Get rid of that change on your branch.